codetransformer/transformers/constants.py

```diff
@@ -49,6 +49,16 @@ class asconstants(CodeTransformer):
     """
     A code transformer that inlines names as constants.
```

Comment:
should we mention that if there are no arguments it inlines the whole builtins namespace?